### DISCUSSION TITLE
I'm often getting Error "Timed out 600 seconds" in Full Node Status.

### DISCUSSION THREAD

**Discussion by RichardRozehnal** [Chia-Network/chia-blockchain#17094] [Support]
Hello,
i have never had problems with farming Chia but last 3 days I'm still getting error "Timed out 600 seconds" in Full Node Status. I updated to version 2.1.3 and the error still occurs. After this error i have to restart GUI and then synchronize but after 1 - 2 hours i get the error again.
I farm on RPi 4 4GB RAM.

In LOG is this error:

2023-12-19T20:51:00.160 daemon __main__                   : WARNING  Ping error to chia_full_node, closing connection.: ConnectionResetError: Cannot write to closing transport

How can i fix it?

thank you

**Comment by BrandtH22** [comment]
Hey @RichardRozehnal , note that we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

As for the recent issues, we have seen the chain at its current max utilization (which is capped at 50% the total max). This has lead to many stressed systems to lag and fall out of sync as they were not well prepared for this level of utilization. 

I have found that stopping chia, deleting the peers.dat file at `~\.chia\mainnet\db\peers.dat`, then starting chia helps to get nodes back into sync as this forces the client to find new peers. Let us know if this also resolves the timed out error, NOTE - I would also reboot the entire device prior to restarting chia to ensure there are no ghost chia processes running.

**Reply by RichardRozehnal** [reply]
Hello,
i deleted the peers.dat file and restarted farm and after about 2 hours i got the same error. Then i deleted the peers.dat file again and restarted Chia GUI and now I'm farming 10 hours with no error. I hope it will work now with no problems. :)
What else should i try to do if the error occurs again? Is it possible to extend the timeout period?

thank you

**Reply by RichardRozehnal** [reply]
Hello,
it's bad. After 30 hours I got the error again. Then I restarted GUI and after 7 hours the same error again. :(

**Reply by BrandtH22** [reply]
Hey @RichardRozehnal ,  note that we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Thank you for the information, what I would do is:
1. stop chia
2. uninstall the current version
3. install the 2.1.4 release candidate from here https://github.com/Chia-Network/chia-blockchain/releases/tag/2.1.4-rc1 (on the last screen of the install check the box that says "add chia to my PATH" if it is available, this helps with step 5 below)
4. rename the config file at `~\.chia\mainnet\config\config.yaml`
5. run `chia init` (let us know if you have questions about running chia commands)
6. start chia
7. readd plot directories (this can be done via the gui or by manually adding them to the config file, if adding to the config file make sure chia is stopped and any data copied to the config is in the correct format as yaml is very picky)
8. readd pool information (this can be done via the gui settings / advanced / resync which resyncs the wallet or by manually adding them to the config file, if adding to the config file make sure chia is stopped and any data copied to the config is in the correct format as yaml is very picky)

Let the release candidate run for a bit and let us know if you experience the same issues

**Reply by RichardRozehnal** [reply]
Hi,
I installed the new version 2.1.4-RC2 6 days ago and since that time I had no errors. I'm farming 24/7. 

thank you for your help

**Reply by BrandtH22** [reply]
Thank you for the confirmation, very glad to hear it is all running smoothly! 
The rc2 is almost identical to the 2.1.4 release so you are safe to remain on the rc2 until the next version is released 👍 

I am going to close this ticket but don't hesitate to open a new one if other issues arise or join us in our discord server (https://discord.gg/chia)

**Reply by RichardRozehnal** [reply]
Hello again,
unfortunatelly, last 10 days I'm getting this error again every day. :( I have installed the latest version 2.2.1.
